Marlin/temperature.cpp

```diff
@@ -1544,70 +1546,79 @@ void Temperature::isr() {
   #endif
 
   #if DISABLED(SLOW_PWM_HEATERS)
```

Comment:
Unfortunately `ENABLED()` cannot be used that way. Only in preprocessor expressions.